addons/stock/product.py

```diff
@@ -402,6 +406,18 @@ def action_view_routes(self, cr, uid, ids, context=None):
         result['domain'] = "[('id','in',[" + ','.join(map(str, route_ids)) + "])]"
         return result
 
+    def onchange_track_all(self, cr, uid, ids, track_all, context=None):
+        warning = {}
+        if ids and track_all:
+            quant_ids = self.pool.get('stock.quant').search(cr, uid, [('product_id.product_tmpl_id','in',ids), ('lot_id','=', False), ('location_id.usage','=', 'internal')])
```

Comment:
Prefer `self.pool["my.model"]` to `self.pool.get("my.model")`, it will avoid some obscure crashes